test/Milestone.test.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+require('chai').use(require('sinon-chai'));
+const Milestone = require('../src/models/Milestone');
+
+describe('Milestone model', () => {
+  const milestone = new Milestone(
```

Comment:
@ronanfitz if you would please also move all this milestone data into a `MilestoneMock.js` as you did with `IssueMock` above, that will be preferred. Thanks!